script/0_DeployL2Contracts.s.sol

```diff
@@ -15,7 +15,6 @@ contract DeployL2Contracts is Script {
 
         vm.startBroadcast(deployerPrivateKey);
 
-        // @review - this is not deployed using a CREATE2 factory as described in README
```

Comment:
foundry scripts use `0x4e59b44847b379578588920ca78fbf26c0b4956c` when you use this syntax

Reply:
I see, can you add a comment?